conn.go

```diff
@@ -135,6 +136,11 @@ func NewConnWith(conn net.Conn, config ConnConfig) *Conn {
 		}},
 	}).size()
 	c.fetchMaxBytes = math.MaxInt32 - c.fetchMinSize
+	var err error
+	c.apiVersions, err = c.ApiVersions()
+	if err != nil {
+		c.apiVersions = nil
```

Comment:
Should we put a default value here?

Reply:
I guess I can put the value that lists all the versions that the current implementation of kafka-go works with.

Reply:
Seems like it could help making good use of this field 👍